### DISCUSSION TITLE
<A-j> Not working in mac?

### DISCUSSION THREAD

**Discussion by byteZorvin** [LazyVim/LazyVim#4262] [Q&A] [answered]
I am not able to get line moving up and down motion in, it used work on my linux but its not working here
How should I check and fix this

**Comment by dpetka2001** [comment]
Most likely terminal issue. Google to see if there are any solutions. Or you could try with `kitty` terminal for example on Mac to see if the issue persists.

**Reply by byteZorvin** [reply]
I am using Alacritty, and have added the following line to the keymaps.lua as well
```
vim.keymap.set("n", "<A-j>", "<cmd>m .+1<cr>==", { desc = "Move Down" })
vim.keymap.set("n", "<A-k>", "<cmd>m .-2<cr>==", { desc = "Move Up" })
vim.keymap.set("i", "<A-j>", "<esc><cmd>m .+1<cr>==gi", { desc = "Move Down" })
vim.keymap.set("i", "<A-k>", "<esc><cmd>m .-2<cr>==gi", { desc = "Move Up" })
```
Anyway way to verify that the issue is alacritty, because I assume there slim chances that could be it

**Reply by dpetka2001** [reply]
No idea about Alacritty. I don't use it. I've tried Kitty and Wezterm and it's working fine on them.

I found the following 2 issues on Alacritty repo https://github.com/alacritty/alacritty/issues/7966 and https://github.com/alacritty/alacritty/issues/93#issuecomment-271043197. Have a read through them to see if you maybe find a solution.

**Reply by byteZorvin** [reply, marked as answer]
Thankyou for redirecting me to the correct place. Found the the solution. Posting here for others.
```
[window]
option_as_alt = "Both"
```
This solved it for me
